brain/python/arrayHelpers.cpp

```diff
@@ -337,12 +339,13 @@ bool gidsFromNumpy(const boost::python::object& object, uint32_ts& result)
     PyArray_Descr* desc = PyArray_DESCR(array);
     msg << "Cannot convert numpy array of type " << desc->kind << desc->elsize
         << " to GID set" << std::endl;
+    // XXX segfaults invalid types (?)
```

Comment:
anything to do code-wise or adding a proper TODO/BUG comment?

Reply:
This is in fact a TODO